client/src/components/AccordianHome.js

```diff
@@ -55,10 +61,13 @@ export default class AccordionHome extends Component {
                 </Accordion.Title>
                 <Accordion.Content active={activeIndex === 2}>
                     <p>
-                        Studies show.....
-                    </p>
-                    <p>
-                        Up to 20% of Americans live in food deserts!
+                        Studies show up to 20% of Americans live in food
+                        deserts. Nearly 2 million Georgia residents,
+                        including about 500,000 children, live in food deserts,
+                        according to an Atlanta Journal Constitution
+                        investigation found <a
+                        href="https://investigations.ajc.com/">
```

Comment:
should we put the actual link to that article here?

Reply:
I think it makes sense to put it here